### DISCUSSION TITLE
Use json file to create multi device on netbox

### DISCUSSION THREAD

**Discussion by chhunnet-leng** [netbox-community/netbox#17429] [General]
Hi guys, anyone have a sample json file to create multi device on netbox, thank you in advance for your kindness help.

**Comment by candlerb** [comment]
Using the [GUI "import"](https://netboxlabs.com/docs/netbox/en/stable/getting-started/populating-data/#bulk-import-csvyaml) function, or using the [REST API](https://netboxlabs.com/docs/netbox/en/stable/integrations/rest-api/#creating-multiple-objects)?

Either way, you provide a JSON list containing one object per device. Here is an example that works for me, with Netbox 4.0.11 and Device Import via the GUI:

```
[{"role":"Server","manufacturer":"Unknown","device_type":"Patch Panel 24","status":"active","site":"Test","name":"foo1"},
{"role":"Server","manufacturer":"Unknown","device_type":"Patch Panel 24","status":"active","site":"Test","name":"foo2"}]
```

**Reply by chhunnet-leng** [reply]
Thank you for your help, I will check.

**Reply by chhunnet-leng** [reply]
I use GUI import
